osu.Game.Tournament/TournamentGameBase.cs

```diff
@@ -64,6 +64,16 @@ private void load(Storage baseStorage)
             Textures.AddStore(new TextureLoaderStore(new StorageBackedResourceStore(storage)));
 
             dependencies.CacheAs(new StableInfo(storage));
+        }
+
+        protected override void LoadComplete()
+        {
+            MenuCursorContainer.Cursor.AlwaysPresent = true; // required for tooltip display
+
+            // we don't want to show the menu cursor as it would appear on stream output.
+            MenuCursorContainer.Cursor.Alpha = 0;
+
+            base.LoadComplete();
 
             Task.Run(readBracket);
```

Comment:
This was moved to `LoadComplete` to allow the new test to exist (I need code to run before this, and it wasn't easy to make happen as long as it was in the `Game`'s async load implementation.

Doesn't seem to have any adverse effects.